nengo_gui/static/netgraph.js

```diff
@@ -335,7 +335,7 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var uid = data.uid;
         for (var i = 0; i < Nengo.Component.components.length; i++) {
             if (Nengo.Component.components[i].uid === uid) {
-                Nengo.Component.components[i].remove(true);
+                Nengo.Component.components[i].remove(true, data.report_back);
```

Comment:
:)  Gah, rereading my comment it does come off as a bit... visceral.  Sorry about that.  

And now that I'm rereading it, I wonder if part of it is that both "report" and "back" have many meanings, so it's unclear without a comment even how to parse this (is "report" a verb or a noun? Is "back" a noun or a verb particle?)  So it's way more potentially confusing than I was initially believing.

So how about ```notify_server```?   That's a little less ambiguous to parse....  :)